src/routes/v2/version_creation.rs

```diff
@@ -95,6 +95,10 @@ pub async fn version_create(
                 json!(legacy_create.game_versions),
             );
 
+            // TODO: This needs to actually get the SideType from somewhere- another version? Default?
+            fields.insert("client_side".to_string(), json!("required"));
+            fields.insert("server_side".to_string(), json!("optional"));
+
```

Comment:
hmm, I think it should just default to the prev version. i would remove this though in this pr bc there is a later linear issue for redoing the side type system

Reply:
Sounds good. The reason i have it in is to keep tests passing rn- as it needs some kind of value in the conversion to call t he route successfully atm. I'll delete them when I merge the side types PR.